packages/jest-mock/README.md

```diff
@@ -98,3 +98,9 @@ In case both `.mockImplementationOnce()` / `.mockImplementation()` and `.mockRet
 
 - if the last call is `.mockReturnValueOnce()` or `.mockReturnValue()`, use the specific return value or default return value. If specific return values are used up or no default return value is set, fall back to try `.mockImplementation()`;
 - if the last call is `.mockImplementationOnce()` or `.mockImplementation()`, run the specific implementation and return the result or run default implementation and return the result.
+
+##### `.withImplementation(function, callback)`
```

Comment:
That’s good to have it here, but for Jest website documentation should added to [docs/MockFunctionAPI.md](https://github.com/facebook/jest/blob/main/docs/MockFunctionAPI.md)

Reply:
Done